client/src/modules/vouchers/complex-voucher.ctrl.js

```diff
@@ -262,6 +268,19 @@ function ComplexJournalVoucherController(
     const voucher = vm.Voucher.details;
     voucher.items = vm.Voucher.store.data;
 
+    /**
+    * FIXE ME
+    * With the use of Journal Vouchers for the payment of the salaries
+    * of the agents, it was question of knowing the identifier
+    * of the payment to facilitate the update of the payment table finally to determine
+    * if the payment is partial or total, and in case of partial payment
+    * it would be impossible to determine the indetifier of the payment,
+    * reason for which we had used the property document_uuid to preserve the identifier of the table
+    */
+    if (voucher.type_id === 7) {
```

Comment:
This code will be easier to modify in the future if you name this number:
```js
if (voucher.type_id === bhConstants.transactionType.SALARY_PAYMENT) {
 // etc..
}
```

Also great comment!